_components/button-groups/01-default.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Default button group
```

Comment:
Total nit-pick, but I prefer consistency in the titles. This one is "Default button group" while the other is simply, "Segmented."
I suggest either including or excluding "button group" in both titles for the subsections.
My preference is to match the titles exactly to the nav, so they would be "Default" and "Segmented."

Reply:
Yeah, I went back and forth on that yesterday but I think I agree with you

Reply:
@Ben-Judy-Flexion This is resolved now